src/ModalBase/Portal.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+class Portal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.rootName = 'paragon-portal-root';
```

Comment:
Does this hard coded root imply we can only have one modal at a time?  Even for ModalPopup?  Or will it append more modals to the same root safely?

Reply:
It appends them safely. @muselesscreator proved this earlier with `Sheet`